osis-core/src/main/java/com/scality/osis/platform/utils/ScalityConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.scality.osis.platform.utils;
+
+public final class ScalityConstants {
+
+    private ScalityConstants() {
+    }
+
+    public static final String TENANT_EMAIL_SUFFIX = "@osis.account.com";
+
+    public static final String ARRAY_SEPARATOR = "_";
+    public static final String ICON_PATH = "/ceph.png";
```

Comment:
we might need to change this.

Reply:
This constant is not required at this moment. Deleting this

Reply:
modified this in commit [068a6b5](https://github.com/scality/vmware-ose-scality/pull/13/commits/068a6b5d2e1d4d6bc2927b57c49c206b7ca53033)